demos/keyboard-nav/index.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Blockly Demo: Keyboard Navigation</title>
+  <script src="../../blockly_compressed.js"></script>
+  <script src="../../blocks_compressed.js"></script>
+  <script src="../../javascript_compressed.js"></script>
+  <script src="../../msg/js/en.js"></script>
+  <style>
+    body {
+      background-color: #fff;
+      font-family: sans-serif;
+    }
+    h1 {
+      font-weight: normal;
+      font-size: 140%;
+    }
+  </style>
+</head>
+<body>
+  <h1><a href="https://developers.google.com/blockly/">Blockly</a> &gt;
+    <a href="../index.html">Demos</a> &gt; Keyboard Navigation</h1>
+
+  <p>You can enter keyboard navigation mode by <b>shift click anywhere on the
+  workspace or on a block.</b> <br>
+  You can then use the WASD keys to move around the workspace or click the show
+  demo button to display an alternative way to navigate the workspace. The demo
+  uses <a href="https://en.wikipedia.org/wiki/Tree_traversal#Pre-order_(NLR)">pre
+  order tree raversal</a> as an alternative way to navigate the blocks,
+  connections, and fields on the workspace.<br><br>
+    W: Previous block/field/input at the same level<br>
+    A: Up one level (Field (or input) -> Block -> Input (or field) -> Block ->
+    Stack -> Workspace)<br>
+    S: Next block/field/input at the same level<br>
+    D: Down one level (Workspace -> Stack -> Block -> Input ( or field) -> Block
+     -> Field (orinput))<br><br>
+  </p>
+
+  <!-- TODO: Update when we add keyboard navigation to site -->
+  <!-- <p>&rarr; More info on <a href="">Keyboard Navigation</a>.</p> -->
+
+  <p>
+    <button onclick="showDemo()">Show Demo</button>
```

Comment:
What about "Show Traversal" (and update other references) instead of "Show Demo"?